src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -511,7 +511,9 @@ static UCS_CLASS_INIT_FUNC(uct_gga_mlx5_iface_t,
                               &init_attr);
 
     status = uct_rc_mlx5_dp_ordering_ooo_init(
-            &self->super, UCT_IB_MLX5_MD_FLAG_DP_ORDERING_OOO_RW_RC,
+            &self->super,
+            ucs_min(md->dv_ooo_cap.max_dp_ordering_rc,
```

Comment:
`(...max_dp_ordering_rc >= OOO_RW)` to get AR enabled boolean passed as parameter?